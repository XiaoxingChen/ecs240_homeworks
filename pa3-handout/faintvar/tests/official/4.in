c Branches
p 3 4 4 1 4
c bb1: ;
b 1
e 1 2
e 1 3
c bb2: v1 = v2;
b 2 1 2
e 2 4
c bb3: v2 = v3;
b 3 2 3
e 3 4
c bb4: print(v1, v2);
b 4 0 1 2
