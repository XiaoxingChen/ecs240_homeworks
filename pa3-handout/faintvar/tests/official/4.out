fvin 1
fvin 2 1 3
fvin 3 2
fvin 4 3
